Source/WebCore/style/StyleResolveForFontRaw.cpp

```diff
@@ -73,7 +73,11 @@ std::optional<FontCascade> resolveForFontRaw(const FontRaw& fontRaw, FontCascade
         switchOn(item, [&] (CSSValueID ident) {
             isGenericFamily = ident != CSSValueWebkitBody;
             if (isGenericFamily)
-                family = familyNamesData->at(CSSPropertyParserHelpers::genericFontFamilyIndex(ident));
+                // FIXME: system-ui need to be treated like a non generic font for the moment
+                if (ident == CSSValueSystemUi)
+                    family = nameString(CSSValueSystemUi);
+                else
+                    family = familyNamesData->at(CSSPropertyParserHelpers::genericFontFamilyIndex(ident));
```

Comment:
nit: Needs to be in braces `{ }` according to webkit style, since `if (isGenericFamily)` spans multiple lines.

Also, the fixme should probably reflect what to do:

// FIXME: Treat system-ui like other generic font families.

Reply:
Fixed